lib/cardano_explorer/base.rb

```diff
@@ -11,13 +11,14 @@ def initialize(hash)
     end
 
     def summary
-      response.parsed_response['Right']
+      @summary ||= get "#{endpoint}/summary/#{@hash}"
```

Comment:
There's already an `attr_reader :hash` so we should use `hash` instead of `@hash` here.

Reply:
Updated. Old habits :)